src/components/ui/EditForm/index.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Button from '../Button';
+import EditIcon from '../EditIcon';
```

Comment:
Path aliasing have not been resolved here.

Reply:
resolved